CI_YAML.md

```diff
@@ -124,33 +125,155 @@ following are a list of keys that are reserved for special use.
 **Properties** is a Map<String, String> and any special values must be JSON encoded
 (i.e. no trailing commas). Additionally, these strings must be compatible with YAML multiline strings
 
-**$flutter/osx_sdk**: xcode configs including sdk and runtime. **Note**: support on legacy `xcode`/`runtime`
-properties and `xcode` dependency has been deprecated.
+<table>
+  <tr>
+    <td>
+      <b>Property Name</b>
+    </td>
+    <td>
+      <b>Description</b>
+    </td>
+    <td>
+      <b>Default Value</b>
+    </td>
+    <td>
+      <b>Type</b>
+    </td>
+    <td>
+      <b>Example</b>
+    </td>
+  </tr>
+  <tr>
+    <td>add_recipes_cq</td>
+    <td>Whether to add this target to flutter/recipes CQ. This ensures
+changes to flutter/recipes pass on this target before landing.
+    </td>
+    <td>"false"</td>
+    <td>string bool</td>
+    <td>
 
-Example:
 ``` yaml
-$flutter/osx_sdk : >-
-  {
-    "sdk_version": "14e222b",
-    "runtime_versions":
-      [
-        "ios-16-4_14e222b",
-        "ios-16-2_14c18"
-      ]
-  }
+add_recipes_cq: "true"
 ```
+</td>
+  </tr>
+  <tr>
+    <td>cache_name</td>
+    <td>The name identifier of the second layer Engine source cache.
+    </td>
+    <td>N/A</td>
+    <td>string</td>
+    <td>
 
-**add_recipes_cq**: String boolean whether to add this target to flutter/recipes CQ. This ensures
-changes to flutter/recipes pass on this target before landing.
+``` yaml
+cache_name: "builder"
+```
+</td>
+  </tr>
+  <tr>
+    <td>cache_path</td>
+    <td>The paths of Engine checkout source that will be auto saved to CAS for boosting source checkout when
+caches no longer exist from the bots.
+    </td>
+    <td>N/A</td>
+    <td>list</td>
+    <td>
+
+``` yaml
+cache_paths: >-
+  [
+    "builder",
+    "git"
+  ]
+```
+</td>
+  </tr>
+  <tr>
+    <td>cache_root</td>
+    <td>The root of cache directory used in recipes: `api.path[cache_root]`.
```

Comment:
I am a little confused by this because there is a well-defined cache_root among the paths that the recipes framework provide and if that is the case we wouldn't need to specify that here unless this is a root within that well-defined directory.

Reply:
This is a good point. The `cache_root` here is exactly the one from recipes world. Added a TODO in https://github.com/flutter/flutter/issues/143671 and removed the entry here.